benchmarks/for.cu

```diff
@@ -13,8 +13,8 @@ void parallel_for(nvbench::state& state) {
   // Build a graph using a sample csr.
   auto csr = io::sample::csr();
   auto G =
-      graph::build::from_csr<memory_space_t::device, graph::view_t::csr>(csr);
-
+      graph::build::build<memory_space_t::device>({}, csr); 
```

Comment:
We can get rid of the namespace `build` because it seems redundant now.

Reply:
Ok this is done!